src/js/tracing/reactnativetracing.ts

```diff
@@ -1,13 +1,9 @@
 /* eslint-disable max-lines */
-import type { Hub } from '@sentry/core';
-import { getCurrentHub } from '@sentry/core';
-import type { IdleTransaction, RequestInstrumentationOptions, Transaction } from '@sentry/tracing';
-import {
-  defaultRequestInstrumentationOptions,
-  getActiveTransaction,
-  instrumentOutgoingRequests,
-  startIdleTransaction,
-} from '@sentry/tracing';
+import type { RequestInstrumentationOptions } from '@sentry-internal/tracing';
+import { instrumentOutgoingRequests } from '@sentry-internal/tracing';
```

Comment:
seems like we need to export this from `@sentry/browser`! With that we can remove `@sentry-internal/tracing` usage as well :)

I'll take care of making that fix.

Reply:
Yes, that would be nice. Thank you.